sdk/provisioning/Azure.Provisioning.Authorization/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
```

Comment:
Do we want this as a separate library or should it be part of the base `Azure.Provisioning`?  I'm leaning toward keeping it in `Azure.Provisioning` because we'll want to add managed identity support in EVERY library we can.

Reply:
Had discussed this with @m-nash previously and the decision was to map the provisioning libraries 1:1 with the ARM libraries, otherwise versioning becomes very messy.

Reply:
Can you or Michael elaborate on the versioning concern?  I'm not clear on why we couldn't ship two or three of the most crosscutting RPs in Azure.Provisioning.

Reply:
I believe the long-term goal here would be to ship updated versions of the ARM libraries with the corresponding Provisioning libraries whenever there is an ARM update. We could still do this even if several RPs were used in Azure.Provisioning, but we would probably need to find a process to make this easy if the packages don't belong to the same service directory. I do agree that it may be cumbersome for customers to need to remember to pull in Azure.Provisioning.Authorization to assign a storage role, for example.

Reply:
I think the trick will be to keep the resource type versions in sync with the Azure.ResourceManager.* libraries.

Right now we are hard coding the default resource type in two places and when we have 1 to 1 mapping with Azure.Provisioning.* this is simpler.

We can also consider meta packages if we wanted to bundle the N most common things so customers didn't have to think / pull in the same 15 packages every time.

Reply:
I'll collapse Auth/ManagedIdentities into the main package for now but we can revisit before GA.